e2e/pages/Onboarding6.po.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
+const buttonLabel = 'Next';
+const screenTitle = 'Header';
```

Comment:
screenTitleTestId here and in onboarding5

otherwise it looks like the pages should actually say Header/Subtitle

Reply:
@tstirrat 
```suggestion
const screenTitleTestId = 'Header';
```

?